scripts/build.py

```diff
@@ -612,16 +733,13 @@ def build_files(source, target, file_hashes, file_formats=None):
             else:
                 # Skip files that are not the specified format.
                 continue
-            tasks.append(task)
-        try:
-            _execute_tasks(tasks)
-        except Exception as e:
-            print e
+            build_tasks.append(task)
+    return build_tasks
 
 
-def rebuild_new_files(
+def generate_build_task_to_rebuild_new_files(
         source_path, target_path, recently_changed_filenames, file_hashes):
-    """Minify recently changed files.
+    """Queue up build task to minify recently changed files.
```

Comment:
task --> tasks (I think?)

Also this should be reflected in the function name.

Reply:
Yep. Fixed.